recipes/mold/1.4.x/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+import os
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.layout import basic_layout
+
+
+class MoldTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    # VirtualRunEnv can be avoided if "tools.env.virtualenv:auto_use" is defined
+    # (it will be defined in Conan 2.0)
+    generators = "VirtualRunEnv"
+    test_type = "explicit"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
```

Comment:
Shouldn't this be a tool requires since we're testing a tool?

Reply:
Yes, I have looked up in ninja